src/options.ts

```diff
@@ -30,4 +30,7 @@ export interface BaseOptions {
 export interface Options extends BaseOptions {
   // TODO(samstern): Remove this once options is better typed
   [key: string]: unknown;
+
+  // whether it's coming from the VS Code Extension
+  isVSCE?: true;
```

Comment:
not sure about this but thought it would be great to add as it's a new option. Didn't want to add it to `BaseOptions` as it will only be available through the extension. Happy to move it to a proper place if needed

Reply:
This seems ok to me as long as it doesn't break anything for the CLI.

Reply:
@joehan Since it will only be present in the options when coming from the extension I think it will always be `true`, otherwise it would be `undefined`. From what I see it can never be `false`, but I can change if needed